.changeset/perfect-tools-notice.md

```diff
@@ -0,0 +1,6 @@
+---
+"docs": patch
+"@espressive/cascara": patch
```

Comment:
@lordpixel this is not a patch change

Reply:
Changed to minor, 👍

Reply:
@lordpixel anything that requires an end user to have to change their code to use our updates is always going to be a major, breaking change. If I as an end user can use this version without changing my code at all, then we know the change is not major. Due to the fact we are renaming components and removing component exports, this is absolutely a breaking change.

@rociobaena @lordpixel we should be aware of these rules here if we are ever unsure what level of change we are making with patch, minor, major: https://semver.org/